src/blocks/button/Button.tsx

```diff
@@ -41,6 +41,7 @@ const StyledButton = styled.button<ButtonProps>`
   justify-content: center;
   white-space: nowrap;
   flex-shrink: 0;
+  flex: 1;
```

Comment:
We cannot add this here as this will affect Button Design which is used in other places.